burger/src/components/Main/Ingridients/reducerIngridients.js

```diff
@@ -8,21 +9,49 @@ export const initialIngridients = {
 };
 
 export function reducerIngridients(state, action) {
+  if (action.type === "clear") {
```

Comment:
It does not work
https://user-images.githubusercontent.com/40695369/208492070-51c46408-c29f-4fa4-a791-1e4a607c1d5d.mov